### PR TITLE
Testing (v1.40, ldap changes)

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

* Upgrade to v1.40
* Move LDAP configuration to json file as recommended

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)
- [x] Upgrade script to check…

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: conf/LocalSettings.php [hunk 1]
```diff
@@ -119,71 +121,42 @@
...
-        "realnameattribute" => "cn",
-        "emailattribute" => "mail"
-      ]
-    ]
-  ];
-
-  return new \MediaWiki\Extension\LDAPProvider\DomainConfigProvider\InlinePHPArray( $config );
-};
-$createaccount = __CREATE_ACCOUNT__;
-$wgGroupPermissions['*']['createaccount'] = $createaccount;
```
Comment (alexAubin) [inline conf/LocalSettings.php:189]: That tweak is needed for the config panel to work tho

Comment (Salamandar) [reply conf/LocalSettings.php:189]: Yeah I renamed the option, I should’ve updated the config panel too…

Comment (Salamandar) [reply conf/LocalSettings.php]: I reverted this change, should be merged at once.

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

**Pull request merged**